lib/easypost/client.rb

```diff
@@ -20,35 +20,39 @@ def initialize(api_key:, read_timeout: 60, open_timeout: 30, api_base: 'https://
 
     # Make an HTTP client once, reuse it for all requests made by this client
     # Configuration is immutable, so this is safe
-    @http_client = EasyPost::HttpClient.new(base_url, http_config)
+    @http_client = EasyPost::HttpClient.new(api_base, http_config)
   end
 
   SERVICE_CLASSES = [
     EasyPost::Services::Address,
     EasyPost::Services::ApiKey,
     EasyPost::Services::Batch,
+    EasyPost::Services::BetaReferralCustomer,
```

Comment:
question: where is the `BetaRate` and `BetaCarrierMetadata` classes? Coming in the next PR?

Reply:
Coming up in the next PR